hrm/department_app/models/employees.py

```diff
@@ -0,0 +1,18 @@
+"""Django models"""
+from django.db import models
+from djmoney.models.fields import MoneyField
+
+
+class Employees(models.Model):
+    """Employees"""
+    Id = models.IntegerField('ID')
+    name = models.CharField('Name',max_length=50)
```

Comment:
The name field should not be null or empty.

Reply:
Fixed.

Reply:
The field name can't be primary key, because it is not unique field (we can have 2 different employees with the same name). Please remove primary_key=True and configure this field so that it is not possible to store NULLs and empty strings in it.